packages/teleterm/README.md

```diff
@@ -156,6 +173,6 @@ Resulting files can be found in `sharedProcess/api/protogen`.
  |  +cluster3             |                                             |
  +------------------------+---------------------------------------------+
 ```
-### PTY communication overview (Renderer Process <=> Shared Process)
+## PTY communication overview (Renderer Process <=> Shared Process)
```

Comment:
I would use `###` here, PTY communication section would be then a part of the `Architecture`.